build/building/drivers/remote.md

```diff
@@ -125,7 +125,7 @@ but this is for illustration purposes.)
     $ docker buildx create \
       --name remote-container \
       --driver remote \
-      --driver-opt cacert=.certs/ca.pem,cert=.certs/client-cert.pem,key=.certs/client-key.pem,servername=... \
+      --driver-opt cacert=${PWD}/.certs/ca.pem,cert=${PWD}/.certs/client-cert.pem,key=${PWD}/.certs/client-key.pem,servername=... \
```

Comment:
I would als like replace `...`  with something more explicit that makes it clear that `...` must be the actual server name, not three dots. Normally I wouldnt mind that much BUT we have that `click to clipboard` button on this section in the docs and it's so easy to run this [without thinking] and get frustrated

Reply:
another thing here is, `mkcert` generates the following directory structure:
```shell
$ tree .certs
.certs
├── SAN
├── buildkit-client-certs.yaml
├── buildkit-daemon-certs.yaml
├── client
│   ├── ca.pem
│   ├── cert.pem
│   └── key.pem
└── daemon
    ├── ca.pem
    ├── cert.pem
    └── key.pem
```

so those TLS cert args should be
```shell
cacert=${PWD}/.certs/client/ca.pem,cert=${PWD}/.certs/client/cert.pem,key=${PWD}/.certs/client/key.pem
```

note how I replaced `-` with `/`. The same thing should be done for the `daemon`